app/assets/javascripts/discourse/app/controllers/tag-show.js

```diff
@@ -92,7 +94,7 @@ export default DiscoverySortableController.extend(
     },
 
     @action
```

Comment:
Given that that the action has been moved into the mixin, I think we can remove this `@action` decorator here which would also make the method/function here consistent with `callResetNew` in the `discovery/topics.js` file.